rskj-core/src/main/resources/reference.conf

```diff
@@ -165,7 +165,7 @@ peer {
 miner {
     # The default gas price
     minGasPrice = 0
-
+    gasPriceCalculatorType = LEGACY
```

Comment:
as this doesn't seem to be that specific to miners, I guess this config should go under the `rpc` section - same as `minGasPriceMultiplier`, wdyt?

Reply:
I was not sure where  to place it, because it is not only affecting the rpc module. Just put it there because we had the min gas price property there, but maybe we could move it to a general section?

Reply:
you mean that it's also used in mem pool? hmm.. what about putting it under the `transaction` section? similar to `transaction.gasPriceBump`